app/src/main/java/nextstep/shoppingcart/component/ProductItem.kt

```diff
@@ -0,0 +1,86 @@
+package nextstep.shoppingcart.component
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.aspectRatio
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.foundation.layout.wrapContentWidth
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+import nextstep.shoppingcart.ui.theme.ShoppingCartTheme
+import java.text.NumberFormat
+import java.util.Locale
+
+@Composable
+fun ProductItem(
+    modifier: Modifier = Modifier,
+    name: String,
+    imageUrl: String,
+    price: Long
+) {
+    val formattedAmount = remember(price) {
+        NumberFormat.getNumberInstance(Locale.KOREA).format(price) + "원"
+    }
+    Column(
+        modifier
+            .wrapContentWidth()
+            .wrapContentHeight()
+    ) {
+        AsyncImage(
+            modifier = Modifier
+                .fillMaxWidth()
+                .aspectRatio(1f),
+            model = imageUrl,
+            contentDescription = name,
+        )
+        Spacer(modifier = Modifier.height(height = 8.dp))
+        Text(
+            modifier = Modifier.padding(start = 4.dp),
+            text = name,
+            style = TextStyle(
+                fontSize = 16.sp,
+                fontWeight = FontWeight.Bold,
+                lineHeight = 14.sp,
+                letterSpacing = 0.5.sp,
+                textAlign = TextAlign.Left
+            ),
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis
+        )
+        Text(
+            modifier = Modifier.padding(start = 4.dp),
+            text = formattedAmount,
```

Comment:
formattedAmount를 remember로 잘 관리해주셨네요. 👍 
하지만 state를 통하여 관리하고 있지는 않은데요. 🤔 
그렇다면, 준보님이 구현하신 코드와 아래 코드는 어떤 차이점이 있을까요?
```suggestion
            text = NumberFormat.getNumberInstance(Locale.KOREA).format(price) + "원",
```